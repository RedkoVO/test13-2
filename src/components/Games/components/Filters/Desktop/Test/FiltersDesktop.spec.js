import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import FiltersDesktop from '../'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <FiltersDesktop />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
