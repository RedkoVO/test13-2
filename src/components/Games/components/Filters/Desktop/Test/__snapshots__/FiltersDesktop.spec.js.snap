// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="FiltersDesktop-root-1"
>
  <div
    className="FiltersDesktop-sortGenre-2"
  >
    <span
      className="FiltersDesktop-sortGenreName-3"
    >
      genre:
    </span>
    <a
      className="FiltersDesktop-sortGenreItem-4"
      href="/"
      onClick={[Function]}
    >
      dice games
    </a>
  </div>
  <div
    className="FiltersDesktop-sortRating-5"
  >
    <span
      className="FiltersDesktop-sortRatingName-6"
    >
      rating:
    </span>
    <a
      className="FiltersDesktop-sortRatingItem-7"
      href="/"
      onClick={[Function]}
    >
      more than 6 stars
    </a>
  </div>
  <div
    className="FiltersDesktop-sortSearch-11"
  >
    <input
      className="FiltersDesktop-sortSearchField-12"
      placeholder="Search..."
      type="text"
    />
  </div>
  <div
    className="FiltersDesktop-sortContract-8"
  >
    <span
      className="FiltersDesktop-sortContractName-9"
    >
      volume of smart contract:
    </span>
    <a
      className="FiltersDesktop-sortContractItem-10"
      href="/"
      onClick={[Function]}
    >
      more than 10.000 CTP
    </a>
  </div>
</div>
`;
